Softeq.XToolkit.WhiteLabel.iOS/Services/StoryboardDialogsService.cs

```diff
@@ -72,6 +74,39 @@ public class StoryboardDialogsService : IDialogsService
             return dialogResult.Task;
         }
 
+        public Task<TEnum> ShowDialogAsync<TEnum>(string title,
+            string message,
+            Dictionary<TEnum, DialogOption> actions)
+            where TEnum : Enum
```

Comment:
Should we really restrict usage to only enums? Seems like any type can be used here

Reply:
I thought about that too, will remove it